app/src/main/java/org/simple/clinic/patient/PatientSearchHelpers.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.patient
+
+/**
+ * [Regex] for stripping patient names and search queries of white spaces and punctuation
+ *
+ * Currently matches the following characters
+ * - Any whitespace
+ * - Comma, Hyphen, SemiColon, Colon, Underscore, Apostrophe, Period
+ * */
+private val spacePunctuationRegex = Regex("[\\s;_\\-:,'\\\\.]")
+
+fun convertNameToSearchableForm(string: String) = string.replace(spacePunctuationRegex, "")
```

Comment:
@vinaysshenoy why did you decide to move these outside of `PatientRepository`?

Asking because I really dislike creating `Utils` and `Helper` classes. I try hard to avoid them 😌

Reply:
It's because I needed to access the function in the migration, and I didn't want to copy the same function/regex in the migration or access the repository from the migration.

Reply:
Maybe a good place to put them would be in the top-level of the `Patient.kt` file since it is the entity.